Core/Inc/monitor.h

```diff
@@ -8,30 +8,43 @@
 #include "pdu.h"
 #include "steeringio.h"
 
-typedef struct {
-	uint16_t brake_value;
-	uint16_t accelerator_value; /* 0-100 */
-} pedals_t;
+/**
+ * @brief Get the debounced TSMS reading.
+ * 
+ * @return State of TSMS.
+ */
+bool get_tsms();
 
-pedals_t get_pedal_data();
-bool get_brake_state();
+typedef struct {
+	mpu_t *mpu;
+	pdu_t *pdu;
+} non_func_data_args_t;
 
-bool get_tsms_reading();
+/**
+ * @brief Task for collecting non functional data from the car. Includes LV battery voltage reading and fuse monitor reading.
+ * 
+ * @param pv_params Pointer to non_func_data_args_t
+ */
+void vNonFunctionalDataCollection(void *pv_params);
```

Comment:
non functional? what is this supposed to mean

Reply:
The data collected in this thread does not effect the function of the car.